ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.19__upgrade.yaml

```diff
@@ -0,0 +1,182 @@
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.24-openshift-4.20
+  openshift_release_rhel-9-release-golang-1.23-openshift-4.19:
+    name: release
+    namespace: openshift
+    tag: rhel-9-release-golang-1.23-openshift-4.19
+  operator-sdk:
+    name: "4.19"
+    namespace: origin
+    tag: operator-sdk
+binary_build_commands: make build
+build_root:
+  from_repository: true
+  use_build_cache: true
+images:
+- dockerfile_path: build/Dockerfile.ci
+  inputs:
+    ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19
+    ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20
+    openshift_release_rhel-9-release-golang-1.23-openshift-4.19:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.23-openshift-4.19
+  to: windows-machine-config-operator-test
+operator:
+  bundles:
+  - as: wmco-bundle
+    context_dir: .
+    dockerfile_path: bundle.Dockerfile
+    skip_building_index: true
+  substitutions:
+  - pullspec: REPLACE_IMAGE
+    with: pipeline:windows-machine-config-operator-test
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.20"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+tests:
+- as: 4-20-upgrade
+  cron: 46 8 * * 0
+  steps:
+    cluster_profile: azure-2
+    test:
+    - as: install
+      cli: latest
+      commands: |
+        export WMCO_NS=openshift-windows-machine-config-operator
+        oc create namespace $WMCO_NS
+        oc label --overwrite=true ns $WMCO_NS openshift.io/cluster-monitoring=true \
+        pod-security.kubernetes.io/enforce=privileged
+
+        operator-sdk run bundle --timeout=15m --security-context-config restricted -n $WMCO_NS "$CURRENT_BUNDLE" \
+        || oc get csv -n $WMCO_NS | awk '{print $1}' | tail -n1 | xargs oc patch csv -n $WMCO_NS --type='json' \
+        -p="[{\"op\": \"replace\", \"path\": \"/spec/install/spec/deployments/0/spec/template/spec/containers/0/image\", \"value\": \"$CURRENT_OPERATOR\"}]"
+
+        sleep 10
+        oc delete deployment -n $WMCO_NS windows-machine-config-operator
+
+        retries=0
+        while ! oc get -n $WMCO_NS deployment windows-machine-config-operator; do
+          if [[ $retries -eq 10 ]]; then
+            echo max retries hit
+            exit 1
+          fi
+          sleep 1m
+          retries=$((retries+1))
+        done
+        oc wait --timeout=10m --for condition=Available -n $WMCO_NS deployment windows-machine-config-operator
+      dependencies:
+      - env: CURRENT_BUNDLE
+        name: wmco-bundle
+      - env: CURRENT_OPERATOR
+        name: current-wmco
+      from: operator-sdk
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 400Mi
+    - as: upgrade
```

Comment:
Some tests need to be ran before the operator is upgraded.
Please look at the existing upgrade tests. The workflow should be the same minus the cluster upgrade.